### ISSUE TITLE
qwen2.5-vl微调过程中forward中的input_ids为None

### ISSUE THREAD

**Issue by rainstorm12** [modelscope/ms-swift#5250]
想基于swift框架轻微改动一下模型结构，训练用的是sft的命令，然后在modelling文件中希望修改一些内容，但是发现input_ids是None：
~~~
NPROC_PER_NODE=2 \
MAX_PIXELS=1003520 \
CUDA_VISIBLE_DEVICES=1,2 \
swift sft \
    --model /.../model_hub/Qwen/Qwen2.5-VL-7B-Instruct \
    --train_type full \
    --dataset /.../sft_format.jsonl \
    --torch_dtype bfloat16 \
    --attn_impl flash_attn \
    --freeze_vit true \
    --freeze_llm false \
    --freeze_aligner false \
    --num_train_epochs 3 \
    --per_device_train_batch_size 1 \
    --per_device_eval_batch_size 1 \
    --learning_rate 1e-5 \
    --gradient_accumulation_steps 2 \
    --eval_steps 100 \
    --save_steps 400 \
    --logging_steps 5 \
    --max_length 4096 \
    --output_dir output/Qwen2_5_VL \
    --dataloader_num_workers 2 \
    --dataset_num_proc 8 \
    --deepspeed zero3 \
~~~

然后在transformer(版本为4.51.3)的modeling_qwen2_5_vl.py的forward函数打印：
~~~
class Qwen2_5_VLForConditionalGeneration(Qwen2_5_VLPreTrainedModel, GenerationMixin):
    _tied_weights_keys = ["lm_head.weight"]
    config_class = Qwen2_5_VLConfig
    _no_split_modules = ["Qwen2_5_VLDecoderLayer", "Qwen2_5_VLVisionBlock"]
..........

    @add_start_docstrings_to_model_forward(QWEN2_5_VL_INPUTS_DOCSTRING)
    @replace_return_docstrings(output_type=Qwen2_5_VLCausalLMOutputWithPast, config_class=_CONFIG_FOR_DOC)
    def forward(
        self,
        input_ids: Optional[torch.LongTensor] = None,
        attention_mask: Optional[torch.Tensor] = None,
        position_ids: Optional[torch.LongTensor] = None,
        past_key_values: Optional[List[torch.FloatTensor]] = None,
        inputs_embeds: Optional[torch.FloatTensor] = None,
        labels: Optional[torch.LongTensor] = None,
        use_cache: Optional[bool] = None,
        output_attentions: Optional[bool] = None,
        output_hidden_states: Optional[bool] = None,
        return_dict: Optional[bool] = None,
        pixel_values: Optional[torch.Tensor] = None,
        pixel_values_videos: Optional[torch.FloatTensor] = None,
        image_grid_thw: Optional[torch.LongTensor] = None,
        video_grid_thw: Optional[torch.LongTensor] = None,
        rope_deltas: Optional[torch.LongTensor] = None,
        cache_position: Optional[torch.LongTensor] = None,
        second_per_grid_ts: Optional[torch.Tensor] = None,
    ) -> Union[Tuple, Qwen2_5_VLCausalLMOutputWithPast]:
        print(input_ids) ##打印input_ids
        output_attentions = output_attentions if output_attentions is not None else self.config.output_attentions
        output_hidden_states = (
            output_hidden_states if output_hidden_states is not None else self.config.output_hidden_states
        )
        return_dict = return_dict if return_dict is not None else self.config.use_return_dict
~~~

这里的input_ids打印出来是None，有友友遇到过这个情况吗

**Comment by Jintao-Huang** [comment]
inputs_embeds是有值的，可以在这里debgu

https://github.com/modelscope/ms-swift/blob/189ce54bc7bb9bcb2635aa5d8bc2e273d7aa073a/swift/llm/template/template/qwen.py#L317

**Comment by rainstorm12** [comment]
找到啦，原来是在这里
